### PR TITLE
fix: Move platform scripts

### PR DESCRIPTION
**Pull request by eriqua**
## Description

Move platform scripts out of the pipeline folder.
This is the easiest solution to avoid triggering all module pipelines in case changes are made to scripts only used by platform pipelines. 
The alternative would be to explicitly exclude platform folder from triggers in all module workflows

[avm.platform.toggle-avm-workflows](https://github.com/eriqua/bicep-registry-modules/actions/workflows/avm.platform.toggle-avm-workflows.yml)

### CODE REVIEW

File: .github/workflows/avm.platform.publish-tag.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
             Write-Output '::group::Publish tagged module to public bicep registry'
 
             # Load used functions
-            . (Join-Path $env:GITHUB_WORKSPACE 'avm' 'utilities' 'pipelines' 'platform' 'Publish-ModuleFromTagToPBR.ps1')
+            . (Join-Path $env:GITHUB_WORKSPACE 'avm' 'utilities' 'platform' 'Publish-ModuleFromTagToPBR.ps1')
```
Comment (eriqua) [inline .github/workflows/avm.platform.publish-tag.yml:51]: Solution under discussion

Comment (eriqua) [reply .github/workflows/avm.platform.publish-tag.yml:51]: Ping @ChrisSidebotham and @AlexanderSehr here for feedback. We have 2 platform PRs incoming

Comment (AlexanderSehr) [reply .github/workflows/avm.platform.publish-tag.yml:51]: ok - jap we still need to have that discussions / brainstorming ;)

Comment (ChrisSidebotham) [reply .github/workflows/avm.platform.publish-tag.yml:51]: I am for this change as discussed offline, happy to have a quick trio sync if we can squeeze it in

Comment (ChrisSidebotham) [reply .github/workflows/avm.platform.publish-tag.yml:51]: @eriqua - How about using an exclusion for the folder path within the original file structure?

e.g
` - !*/platform/*`

**Comment by ChrisSidebotham** [comment]
@eriqua - can we add this to the loop for the next discussion?

**Comment by ChrisSidebotham** [comment]
Coverting this to draft until a decision/conflict is made... @eriqua @AlexanderSehr

**Comment by ChrisSidebotham** [comment]
@eriqua - Will we be picking this up again soon?

**Comment by eriqua** [comment]
Closing in favor of #1294

**Pull request closed without merging**